spec/lib/percy/client/resources_spec.rb

```diff
@@ -57,8 +57,7 @@
       it { is_expected.to_not eq(other) }
       it { is_expected.to_not eql(other) }
       it { expect(resource.hash).to_not eq(other.hash) }
-
-      it('makes their array unique') { expect([resource, other].uniq).to eq([resource, other]) }
```

Comment:
Are you sure? Rubocop fails this with:

```
spec/lib/percy/client/resources_spec.rb:60:31: E: unexpected token tLCURLY
(Using Ruby 2.1 parser; configure using TargetRubyVersion parameter, under AllCops)
      it 'makes array unique' { expect([resource, other].uniq).to eq([resource, other]) }
                              ^
```

I guess because it would not parse in Ruby 2.1 properly and actually needs the parenthesis around the string before calling the block? Otherwise it would call the block on the string.